filter.go

```diff
@@ -260,6 +260,40 @@ func NewSkbEditAction() *SkbEditAction {
 	}
 }
 
+type PoliceAction struct {
+	ActionAttrs
+	Rate            uint32 // in byte per second
```

Comment:
struct flow_action_entry in kernel defines rate as u64 field rate_bytes_ps.

Reply:
struct tc_ratespec I referred to in kernel defines rate as u32 field.
```c
struct tc_ratespec {
        unsigned char   cell_log;
        __u8            linklayer; /* lower 4 bits */
        unsigned short  overhead;
        short           cell_align;
        unsigned short  mpu;
        __u32           rate;
};
```
And the c library iproute2 also regards the rate field as u32 value.

So which struct should I refered to?

Reply:
do you have equivalent iproute2 command example for this API that may help.

Reply:
Well, I have test it with the following iproute2 command.
```bash
tc qdisc add dev eth0 ingress
tc filter add dev eth0 parent ffff: u32 match u32 0 0 action police rate 1000000kbit burst 100mbit mtu 64k drop
```
And the second patch I sent provides a test about this API.